docs/source/data_acquisition/AzureBlobStorage.rst

```diff
@@ -6,9 +6,10 @@ Description
 
 Azure Blob Storage provides a simple and flexible way to store and access data of any kind.
 This makes it ideal for storing a range of data relating to security investigations, whether
-it be raw data to analyse or to store outputs and findings.
+it be raw data to analyze or to store outputs and findings.
```

Comment:
You've been in the US too long ;)

Reply:
haha
Yeah - just following the wavy lines :-)